ADD_BLOG_ENTRY.md

```diff
@@ -0,0 +1,76 @@
+## Adding a blog entry
+
+Blog entries can be added by following these steps:
+
+- If submitting for the first time an [author](#blog-author) YAML file must be added to the `data/blog/authors` directory.
+- Add an MD file containing the [blog post's metadata and body](#blog-post-content).
+  - each file must be added in the current year's directory, found in `data/blog/`.
+
+> All submissions must follow the [contributing guidelines](https://github.com/laminas/.github/blob/main/CONTRIBUTING.md) as well as the [Code of Conduct](https://github.com/laminas/.github/blob/main/CODE_OF_CONDUCT.md)
+
+### Blog post content
+
+Each blog post **must** begin with the YAML section found in the [template](#template-of-blog-yaml-section).
+
+The `<!--- EXTENDED -->` tag is used to mark a preview section of the blog post on the listing page, with all following content being hidden.
+
+> **Note** that markdown and yaml files are linted as part of the repository's continuous integration pipeline.
+
+### Blog author
+
```

Comment:
I suggest you add an example just to ensure consistency and to make it easier for users.

Reply:
Yes I saw it. Good with me.